### PR TITLE
2 player end game sequence

### LINKED ISSUE #86: End game screen shows whether bob or bub won

### LINKED ISSUE #87: Separate single and double player high scores

### LINKED ISSUE #92: Style buttons, remove bootstrap

### PR DESCRIPTION
**Pull request by adriennedomingus**
To add to the redundancy....

Different end screens based on outcome
Can start new single or double player game from end screen
Adjusted placement of scores and lives to make more visible
High Scores are added to correct section
Added high scores list for 2 player game
Styled buttons with CSS to remove Bootstrap

closes #92 closes #87 closes #86

### CODE REVIEW

**Comment by erinnachen** [comment]
Cool. There's a bug where the scores aren't reset if you move from one player to two player mode.

**Comment by adriennedomingus** [comment]
@erinnachen whoops. Solved.

**Comment by adriennedomingus** [comment]
@erinnachen There appears to be another bug where the high score isn't recorded after restarting the game from the end screen. Changing it back to WIP and will let you know when it's done

**Comment by adriennedomingus** [comment]
JK, it's fixed now.

File: index.html [hunk 1]
```diff
@@ -16,32 +15,68 @@
...
-        <h5 id="instructions" class="btn btn-default">Instructions</h2>
+        <h5 id="start" class="button">Start game: single player</h2>
+        <h5 id="start-double" class="button">Start game: two player</h2>
+        <h5 id="instructions" class="button">Instructions</h2>
       </div>
       <div id="instructions-screen" class="hidden">
         <img src="images/bubble_bobble_logo.png" alt="bubble bobble logo" class="small-logo"/>
         <h3>Instructions</h3>
-        <h5 id="back" class="btn btn-default">Back</h2>
+        <h5 id="back" class="button">Back</h2>
```
Comment (erinnachen) [inline index.html]: I just noticed that the opening and closing h tags don't match

Comment (adriennedomingus) [inline index.html]: Sure enough. Now they do.

**Comment by erinnachen** [comment]
YAY. let's get on battle mode.

**Pull request merged**